src/cfitsio.jl

```diff
@@ -1,5 +1,109 @@
 using Compat
 
+# functions to convert to C integer types (this could be in Julia Base module):
```

Comment:
These conversion functions are fine with me for now, but note that this wouldn't be accepted into Julia Base because conversion functions have been changed in Julia v0.4. Functions like `int32()` have been removed in favor of `Int32(x)` for converting scalars and `Array{Int32}(x)` for converting arrays.

(Eventually we should switch to the new syntax and use Compat to keep supporting v0.3, but that's a different PR!)